miio/integrations/vacuum/mijia/g1vacuum.py

```diff
@@ -373,3 +374,22 @@ def consumable_reset(self, consumable: G1Consumable):
     def set_fan_speed(self, fan_speed: G1FanSpeed):
         """Set fan speed."""
         return self.set_property("fan_speed", fan_speed.value)
+
+    @command()
+    def fan_speed_presets(self) -> Dict[str, int]:
+        """Retrieve predefined fan speeds, dictionary where.
+
+        - key is name (identifier)
+        - value is integer representation usable as argument for set_fan_speed_preset() method
```

Comment:
No need to copy&paste the docstring from the interface, something like this would be enough:
```
"""Return predefined fan speeds."""
```